utils/codegen/codegen-py/asn1CodeGenerationUtils.py

```diff
@@ -709,6 +761,30 @@ def asn1TypeToJinjaContext(asn1_type_name: str, asn1_type_info: Dict, asn1_types
                 "ros_value": size
             })
 
+    # class types
+    elif ".&" in asn1_type_type and asn1_type_type.split(".")[0] in asn1_classes:
+        asn1_class_name = asn1_type_type.split(".")[0]
+        asn1_class = asn1_classes[asn1_type_type.split(".")[0]]
+        for member in asn1_class["members"]:
+            if member["name"] == asn1_type_type.split(".")[1]:
+                class_member = {
+                    "name": asn1_type_info["name"],
+                    "type": member["type"]
+                }
+                if member["type"] not in asn1_types:
+                    class_member["type"] = "CHOICE"
+                    class_member["identified_by"] = "container_id.value" # TODO: get from table
+                    class_member["members"] = []
+                    for value in asn1_values:
+                        for member in asn1_class["members"]:
+                            if member["type"] == asn1_values[value]["type"]:
+                                class_member["members"].append({
+                                    "name": validRosType(value), # TODO: misuse of validRosType; just want to start with a capital letter
+                                    "type": validRosType(value) # TODO: misuse of validRosType; just want to start with a capital letter
```

Comment:
Where is this used and hwy is capital letter needed? If it's only about the capital letter, use `value.capitalize()`?

Reply:
`capitalize()` doesn't work. `"myString".capitalize() -> "Mystring"`